<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="2">
     <widget class="QCheckBox" name="to_flash">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>to_flash</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Reset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QCheckBox" name="coldboot_required">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>coldboot_required</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Flash</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="LoadFlash">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SaveFlash">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Value:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="value"/>
       </item>
       <item>
        <widget class="QPushButton" name="setValue">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>SetValue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetValue">
         <property name="text">
          <string>ResetValue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="6">
     <widget class="QTreeWidget" name="treeView">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QCheckBox" name="is_directory">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>is_directory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>value</sender>
   <signal>returnPressed()</signal>
   <receiver>setValue</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
